src/js/apps/patients/patient/flow/flow_app.js

```diff
@@ -286,19 +288,17 @@ export default SubRouterApp.extend({
   },
 
   showActionSidebar(flowId, actionId) {
-    const actionApp = this.getChildApp('action');
+    const sidebarApp = this.getChildApp('actionSidebar');
 
-    this.listenToOnce(actionApp, {
-      'start'(options, action) {
-        this.setState('actionBeingEdited', action.id);
-        action.trigger('editing', true);
-      },
-      'stop'() {
-        this.setState('actionBeingEdited', null);
-      },
-    });
+    sidebarApp.stop();
 
-    this.startChildApp('action', { actionId });
+    const action = Radio.request('entities', 'actions:model', actionId);
+
+    Radio.request('sidebar', 'start', sidebarApp, { action });
+
+    this.listenTo(sidebarApp, 'close', () => {
```

Comment:
@nmajor25 now this `close` event _always_  indicates the user manually closed the sidebar